lib/ui/discover/discover_viewmodel.dart

```diff
@@ -1,33 +1,33 @@
 import 'package:farmsmart_flutter/data/model/article_entity.dart';
+import 'package:farmsmart_flutter/data/model/articles_directory_entity.dart';
 import 'package:farmsmart_flutter/model/loading_status.dart';
 import 'package:farmsmart_flutter/redux/app/app_state.dart';
 import 'package:farmsmart_flutter/redux/home/discover/discover_actions.dart';
 import 'package:redux/redux.dart';
 
 class DiscoverViewModel {
   LoadingStatus loadingStatus;
-  final List<ArticleEntity> articlesList;
+  final ArticlesDirectoryEntity articleDirectory;
   final ArticleEntity selectedArticle;
 
-  final Function fetchArticles;
-  final Function(ArticleEntity articleData) goToArticleDetail;
+  final Function fetchArticleDirectory;
+  final Function(ArticleEntity articleData) goToDetail;
 
   DiscoverViewModel(
       {this.loadingStatus,
-      this.articlesList,
+      this.articleDirectory,
       this.selectedArticle,
-      this.fetchArticles,
-      this.goToArticleDetail});
+      this.fetchArticleDirectory,
+      this.goToDetail});
 
   static DiscoverViewModel fromStore(Store<AppState> store) {
     return DiscoverViewModel(
-      loadingStatus:  store.state.discoverState.loadingStatus,
-      articlesList: store.state.discoverState.articleList,
-      selectedArticle: store.state.discoverState.selectedArticle,
-      goToArticleDetail: (ArticleEntity article) =>
-          store.dispatch(GoToArticleDetailAction(article)),
-      fetchArticles: () =>
-          store.dispatch(FetchArticleListAction())
-    );
+        loadingStatus: store.state.discoverState.loadingStatus,
+        articleDirectory: store.state.discoverState.articlesDirectory,
+        selectedArticle: store.state.discoverState.selectedArticle,
+        goToDetail: (ArticleEntity article) =>
+            store.dispatch(GoToArticleDetailAction(article)),
+        //TODO: Check
```

Comment:
What does this comment do?

Reply:
Sorry, I forgot this comment. Now it's deleted.